lib/src/components/NxDrawer/NxDrawer.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState
+} from 'react';
+import classnames from 'classnames';
+
+import AbstractDialog from '../AbstractDialog/AbstractDialog';
+import NxCloseButton from '../NxCloseButton/NxCloseButton';
+import withClass from '../../util/withClass';
+
+import { Props, NxDrawerHeaderProps, NxDrawerContextValue, OpenState, propTypes } from './types';
+
+import './NxDrawer.scss';
+
+const NxDrawerContext = React.createContext<NxDrawerContextValue>({
+  closeDrawer: () => {},
+  open: false
```

Comment:
This is unused now right?

Reply:
> This is unused now right?

Yup, removed it here: 2508d2055fe9fa4522c6abf9c565932da2272ebd